src/components/screens/ScoreSetCreator.vue

```diff
@@ -1796,24 +1759,54 @@
                         </template>
                       </FileUpload>
                     </span>
-                    <span v-if="validationErrors.scoresFile" class="mave-field-error">{{
-                      validationErrors.scoresFile
-                    }}</span>
+                    <span v-if="validationErrors.scoresFile" class="mave-field-error">{{validationErrors.scoresFile}}</span>
+                  </div>
+                </div>
+                <div class="mavedb-wizard-row">
+                  <div class="mavedb-wizard-help">
+                    <label :for="scopedId('input-scoreColumnsMetadataFile')">Load a scores column metadata file</label>
+                    <div class="mavedb-help-small">
+                      This file is optional, but recommended.
```

Comment:
I guess I wonder if we might include some additional details here about the required structure of the metadata file. For instance: 
> This file is optional, but recommended. If provided, it should be a JSON file containing a single object. The keys of that object should be limited to columns of the score data, while values should include a string `description` and an optional string `details`. 

I'm not really in love with that description and the answer is probably that it should be added to the documentation, but I think we realistically haven't updated it in so long that bringing the docs up to date is a totally separate issue.